external-crates/move/crates/move-compiler/tests/move_2024/parser/positional_struct_fields_with_idents.exp

```diff
@@ -1,8 +1,15 @@
+error[E03004]: unbound type
+  ┌─ tests/move_2024/parser/positional_struct_fields_with_idents.move:4:23
+  │
+4 │     public struct Foo(x: u64) has copy, drop;
+  │                       ^ Unbound type 'x' in current scope
```

Comment:
Could we carve out a special case for these where, if we find `x: u64` we report an error that we can't name positional fields, and then return _just_ to `u64`?